Makefile

```diff
@@ -0,0 +1,16 @@
+default: help
+
+bootstrap: env.d/local ## Initialize Docker image and environment
+	${MAKE} build;
```

Comment:
Why not depending on the `build` rule?

Reply:
I removed the `bootstrap` rule and kept only `build`.